### DISCUSSION TITLE
info plugin dhcp server hassio

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#752] [Networking]
Buongiorno,
volevo avere qualche info sul plugin presente nei componenti aggiuntivi di hassio, in particolare quello dhcp server.
Volevo capire a cosa potrebbe servirmi. Ho provato a configurarlo e ad impostare delle reservations dei miei hosts presenti nella mia rete, però non capisco come fargli utilizzare al mio router questo dhcp server presente in hassio.
Mi potete aiutare?

**Comment by snakuzzo** [comment]
non capisco il perché di tutto ciò.
Comunque...immagino tu voglia disabilitare il DHCP Server del tuo router ed usare questo addon, corretto? 

Il mio consiglio è... non farlo

**Reply by federico89mi** [reply]
ah ok, grazie del consiglio, giusto per curiosità mi sai spiegare il motivo?

**Reply by federico89mi** [reply]
è solo per il fatto che se mi crolla il raspberry poi rimango fregato giusto?

**Reply by snakuzzo** [reply]
No...non è solo quello.
Il discorso è...perché non farlo fare al router?

**Reply by federico89mi** [reply]
non lo so, mi chiedevo cosa serviva quel plugin. In più se dovessi cambiare router come penso che farò tra qualche mese non dovrò riportarmi tutte le reservations

**Reply by snakuzzo** [reply]
A parte il fatto che non è detto che ti occorrano tutte...è da vedere
Ma il punto poi è... se vuoi farlo, perché usare un addon di HassOS ? 😅
